.gitignore

```diff
@@ -9,3 +9,6 @@ libexec/desktop/hooks/*
 *.swp
 *.swo
 *.idea*
+spec/examples.txt
+Gemfile.lock
```

Comment:
Can you remove the entry for `Gemfile.lock` and add any changes please.  `Gemfile.lock` is part of the process that ensure that the same set of libraries used for developing the application are also used when packaging it into an RPM.

`Gemfile.lock`s are commonly committed for applications, but are usually not committed for libraries.